### ISSUE TITLE
[Translate_pt_BR] Beginners Course intro

### ISSUE THREAD

**Issue by fellyph** [ampproject/amp.dev#3263]
Hi @CrystalOnScript 

I would like to translate the Beginners course introduction to Brazilian Portuguese:   https://github.com/ampproject/amp.dev/blob/future/pages/content/amp-dev/documentation/courses/beginning-course/course-introduction.md

Estimation: 1 week

Thanks,
Fellyph

**Comment by CrystalOnScript** [comment]
That'd be great! We do not currently have this out for translation, so this would be very much appreciated :) 

cc: @morsssss

**Comment by morsssss** [comment]
Yay @fellyph ! This... would be huge.

While we're at it, can you share any thoughts on how to make it easier for students who want to skip ahead past most of the "What is AMP" material and just start building something?

**Comment by fellyph** [comment]
@CrystalOnScript @morsssss 

I can start with the documentation and after translate the slides to be used by GDG groups.

About the workshop, I had done the course at 4 GDG Meetups and 3 DevFest. Some feedback about the process.

The first Workshop was 2.5 hours:
https://www.meetup.com/Google-Developer-Group-Belfast/events/264164052/

Agenda:
-Why AMP?
-How AMP helps
-Requirements
-Starting our journey
-Validation and AMP Cache
-Exercise 1: Using AMP validator
-The anatomy of an AMP Page
-Exercise 2: Converting our HTML Solution
-Web Components?
-Exercise 3: First Components
-Exercise 4: Embedding Video
-Exercise 5: Creating an image SlideShow
-Exercise 6: Adding Sharing Links
-Exercise 7: Building a slide menu

At the first workshop, I've decided to use git and let the students use their IDEs, but I had problems with students that didn't have an IDE or are not familiar with GitHub. The first time I spent half training on the introduction and explaining the first exercise. The second half was in a rush because I had 1 hour to do 7 exercises. In the end, we finished the exercise 5 and explained the theory over 6 and 7.

The second workshop, I've moved to Glitch and reduced the introduction for 40 min, I've finished the 7 exercises without time for question in the end.

The next one I was confident because the previous workshop went well. got at the building 10 min before the training, first, the network firewall at the venue blocked Glitch, my code to start the project on glitch was with the final version, I tried to get the raw version from Git also was blocked for guests. Luckily I had plan B on my machine and we started the course. I had reduced the intro for 20min with two reasons why one more "technical" and another more "business". The agenda was a bit tide but in the end, had all exercises.

After those three workshops, I got the right pace, about setup and timing:
20 min intro
80 min exercises
20 min questions and buffer

I had the checklist:
Clean the exercise files on Glitch
Remind the group that is a hands-on training they need to bring a computer  
Check with the venue if the network has some restriction
Check if the access for the internet needs some extra steps
Arrive 30 min earlier to test the venue, screens, network
Do a quick tutorial about glitch how to remix a project and if they get lost how the can remix again my project to follow the exercise
Show the materials in the end materials, slide and Youtube course
